sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3611,6 +3611,20 @@ object functions {
    */
   def schema_of_json(e: Column): Column = withExpr(new SchemaOfJson(e.expr))
 
+  /**
+   * Parses a column containing a JSON string and infers its schema using options.
+   *
+   * @param e a string column containing JSON data.
+   * @param options JSON datasource options that control JSON parsing and type inference.
```

Comment:
maybe you can say refer to DataFrameReader.json for the list of options

Reply:
Sure but not all options can impact on schema inferring. @rxin Maybe it makes sense to list the options which I pointed out in the ticket: https://issues.apache.org/jira/browse/SPARK-25447 ?